hub-templates/basehub/values.yaml

```diff
@@ -150,6 +150,14 @@ jupyterhub:
     memory:
       guarantee: 256M
       limit: 1G
+    cpu:
+      # When limit is set but not guarantee, guarantee is set to match limit! For most
+      # use cases, when we set limit but not guarantee, we want to offer no guarantee and
+      # a limit. This doesn't seem to be possible at all (need to investigate why). In the
+      # meantime, setting a super low guarantee here means we can guard against issues like
+      # https://github.com/2i2c-org/infrastructure/issues/790, where setting the limit
+      # but not guarantee just gave users a huge guarantee, causing node spin ups to fail
+      guarantee: 0.01
```

Comment:
This can make sense in a way, but it also forces everyone to learn about 2i2c specific configuration.

Its a trade off between that users can assume k8s native behavior vs that they can't and potentially avoids mistakes due to it.

I'm conflicted about setting various defaults in the base chart overall due to this, but since we already have defaults about resource requests/limits I don't see it as an escalation of such complexity.

Reply:
> I'm conflicted about setting various defaults in the base chart overall due to this, but since we already have defaults about resource requests/limits I don't see it as an escalation of such complexity.

+1. Specifically, we already set default memory requests and limits here.